### ISSUE TITLE
Have proper /tmp cleanup

### ISSUE THREAD

**Issue by tituspijean** [YunoHost-Apps/flarum_ynh#126]
See this https://github.com/YunoHost-Apps/flarum_ynh/issues/122#issuecomment-533995924

```
2019-09-23 09:14:12,857: INFO - [##++++..............] > Composer is installing Flarum and its dependencies (may take a while)... [00h00m,01s]
2019-09-23 09:14:12,958: DEBUG - + ynh_exit_properly
2019-09-23 09:14:12,959: WARNING - [37;41m                                                [39;49m
2019-09-23 09:14:12,960: DEBUG - + local exit_code=1
2019-09-23 09:14:12,960: WARNING - [37;41m  [InvalidArgumentException]                    [39;49m
2019-09-23 09:14:12,961: DEBUG - + '[' 1 -eq 0 ']'
2019-09-23 09:14:12,962: WARNING - [37;41m  Project directory /tmp/flarum/ is not empty.  [39;49m
2019-09-23 09:14:12,963: DEBUG - + trap '' EXIT
2019-09-23 09:14:12,963: WARNING - [37;41m                                                [39;49m
```

We need to make sure that `/tmp/$app` is deleted after successful install and after removal.

**Comment by tituspijean** [comment]
Everything seems in order though...

In `remove` script:
https://github.com/YunoHost-Apps/flarum_ynh/blob/aca4ac9746c95f408ee2319b96cfbfbe184bda65/scripts/remove#L84-L86

In `install` script, after installing everything in the temp folder:
https://github.com/YunoHost-Apps/flarum_ynh/blob/aca4ac9746c95f408ee2319b96cfbfbe184bda65/scripts/install#L134-L136

Do you have the install log of the attempt before the one from which you have shown the logs?

**Comment by tituspijean** [comment]
Fixed by e25dddecb68814fa526d742655e8b8322a924745
